src/testdir/test_increment.vim

```diff
@@ -796,4 +796,54 @@ func Test_increment_special_char()
   call assert_beeps("normal \<C-X>")
 endfunc
 
+" Try incrementing/decrementing a number when nrformats contains unsigned
+func Test_unsigned()
```

Comment:
I think should name including `increment` same as other tests; e.g. Test_increment_unsigned or Test_unsinged_increment